### ISSUE TITLE
I only see □□□□□□□□□□□□□ instead of status and compiler messages and dokuments in side bar

### ISSUE THREAD

**Issue by hlech65730** [geany/geany#2576]
hello @ all,
i have a problem with Geany 1.36 under Windows 10. I got only □□□□□□□□□□□□□ in the sidebar and in the status and compiler message window. Only in Windows ! In Raspbian/Debian and on Mac OS its ok ! 
Thanks ! Regards, Hlech65730
![Geany_Problem](https://user-images.githubusercontent.com/34654103/90514523-966ca000-e161-11ea-8aa3-718b40ce866b.jpg)

**Comment by elextr** [comment]
Seems like the default font used for those elements is bad, you could try a different one.

**Comment by rubyFeedback** [comment]
Hmm. If the above hint by elextr does not work, perhaps have a look at pango; I remember having similar issues when I messed up pango-related stuff. Although my guess may be totally wrong since I use Linux (though I think geany needs gtk+pango+atk on windows just as well). I did, however had, test geany some months ago on windows, some already compiled one by someone else, and had no such issue at all, it worked out of the box. (I actually tested that to see if I still should use notepad++ or not, but geany seems to work too so perhaps I may not need notepad++ on windows. Notepad++ is quite decent though, as an editor.)
